nrich-validation/src/main/java/net/croz/nrich/validation/constraint/validator/SpelExpressionValidator.java

```diff
@@ -30,24 +34,24 @@
 
 public class SpelExpressionValidator implements ConstraintValidator<SpelExpression, Object> {
 
-    private String spelExpression;
+    private static final String ENVIRONMENT_VARIABLE_NAME = "environment";
+
+    private static final String SYSTEM_PROPERTIES_VARIABLE_NAME = "systemProperties";
```

Comment:
I think we should cover this part with tests in the same way it's done for the environment.

Reply:
Added.